core/tests/test_utils.py

```diff
@@ -109,15 +109,21 @@ class TestBase(unittest.TestCase):
     # Usernames containing the string 'admin' are reserved, so we use 'adm'
     # instead.
     ADMIN_USERNAME = 'adm'
+    ADMIN_ROLE = feconf.ROLE_ADMIN
     MODERATOR_EMAIL = 'moderator@example.com'
+    MODERATOR_ROLE = feconf.ROLE_MODERATOR
     MODERATOR_USERNAME = 'moderator'
     OWNER_EMAIL = 'owner@example.com'
+    OWNER_ROLE = feconf.ROLE_COLLECTION_EDITOR
```

Comment:
Actually I think you should keep the roles separate from the users. It's confusing for something named ROLE_VIEWER to really be 'editor'.

You don't need to define them here at all; just refer to feconf.ROLE_ADMIN etc. at the relevant places in the tests.